### DISCUSSION TITLE
Why should I not use this to back up my vault?

### DISCUSSION THREAD

**Discussion by Sprocketer** [ObsidianPublisher/obsidian-github-publisher#330] [Q&A — Plugin] [answered]
Found this warning in the plugin:

```
Warning
Do not use this plugin to sync or save your Obsidian Vault!
Avoid opening the converted files from your repository in Obsidian!
```

**Comment by Mara-Li** [comment, marked as answer]
Because the plugin does some edit to your file, and it can create some cyclic edition and broke path with Obsidian

**Reply by Sprocketer** [reply]
Found YAOS (stands for Yet Another Obsidian Synchronizer) plugin for this - thanks!
